Dockerfile

```diff
@@ -190,8 +190,9 @@ USER root
 RUN mkdir -p /usr/share/licenses/aws-iam-authenticator && \
     chown -R builder:builder /usr/share/licenses/aws-iam-authenticator
 
-ARG AWS_IAM_AUTHENTICATOR_VERSION=0.5.3
-ARG AWS_IAM_AUTHENTICATOR_SOURCE_URL="https://github.com/kubernetes-sigs/aws-iam-authenticator/archive/refs/tags/v${AWS_IAM_AUTHENTICATOR_VERSION}.tar.gz"
+ARG AWS_IAM_AUTHENTICATOR_VERSION=0.5.5
+ARG AWS_IAM_AUTHENTICATOR_SHA512_SUM=24c2a7fd141f921e18c17f4c7e90bbf5325cc6fad3b35753c0da2d879cf64d8f6af0130f8c45ec7693d6fce043181abbd04a911e6fa778f686bd5c313631ed0f
+ARG AWS_IAM_AUTHENTICATOR_SOURCE_URL="https://cache.bottlerocket.aws/aws-iam-authenticator-0.5.5.tar.gz/${AWS_IAM_AUTHENTICATOR_SHA512_SUM}/aws-iam-authenticator-${AWS_IAM_AUTHENTICATOR_VERSION}.tar.gz"
```

Comment:
I don't have strong feelings about this, but should we take the `sdk-fetch`/hashes approach that we're using with the other Bottlerocket containers?

Reply:
Hmm, probably, I would need to find a prototype example. Are these in the Bottlerocfket rpm spec files?

Reply:
This is probably the most straight forward example: https://github.com/bottlerocket-os/bottlerocket-admin-container/pull/35

Reply:
I'm going to leave sdk-fetch as a future enhancement. It seems like it makes the most sense when you have all of your sources in the lookaside cache.